mxcube3/ui/components/SampleView/SampleImage.jsx

```diff
@@ -46,6 +47,7 @@ export default class SampleImage extends React.Component {
     this.toggleGridVisibility = this.toggleGridVisibility.bind(this);
     this.canvas = {};
     this.drawGridPlugin = new DrawGridPlugin();
+    this.drawGridPlugin.setGridResultFormat(config.use2dCenteredPoints);
```

Comment:
Should this be:

... setGridResultFormat(config.meshResultFormat);

?
Otherwise I'm confused on relation between 'meshResultFormat' and 'use2dCenteredPoints' configs.

Reply:
No, you are right just me passing the wrong option